.pylintrc

```diff
@@ -2,7 +2,7 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code
-extension-pkg-whitelist=numpy,scipy,autograd,toml,appdir,autograd.numpy,autograd.numpy.linalg,autograd.numpy.builtins,semantic_version,torch,tensorflow,tensorflow.contrib,tensorflow.contrib.eager,LazyLoader,networkx,networkx.dag
+extension-pkg-whitelist=numpy,scipy,autograd,toml,appdir,autograd.numpy,autograd.numpy.linalg,autograd.numpy.builtins,semantic_version,torch,tensorflow,tensorflow.contrib,tensorflow.contrib.eager,LazyLoader,networkx,networkx.dag,catalyst.utils.wrapper
```

Comment:
@dime10 we had a discussion about this yesterday https://pylint.pycqa.org/en/2.3.0/technical_reference/c_extensions.html#pylint-and-c-extensions

Reply:
Although, it is deprecated. Maybe we can indeed delete this.

Reply:
Depends on your definition of deprecation. But yes, `allow-list` should keep the intended meaning, but it also doesn't appear to provide anything for us at the moment, which is why @dime10 also seems to be in favour of removing it. See here https://github.com/PennyLaneAI/catalyst/pull/108/commits/a5077ec6e340ad533bffa48d66df47e04f10db3a

Reply:
Did you confirm that it was indeed required for this PR? If so there is no problem keeping this flag, but yeah my other PR removes all the other entries from it because they were unnecessary.

Reply:
Have you tried adding `catalyst.utils.wrapper` to `ignored-modules`? Not sure it would work but it might

Reply:
We would need to add `catalyst.utils` to `ignored-modules` for this to work which I think has the effect of ignoring all other modules in `catalyst.utils`. I think that the best alternative is just to keep the pylint pragma.

Reply:
@dime10 I'll keep this change here, but I suppose that you can just delete this line in your other PR.

Reply:
Sounds good 👍